### PR TITLE
[MODINVOICE-535] - Release v5.8.0

### PR DESCRIPTION
**Pull request by azizbekxm**
## Purpose

https://folio-org.atlassian.net/browse/MODINVOICE-535
## Pre-Merge Checklist:
Before merging this PR, please go through the following list and take appropriate action.

- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code are 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added, or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all the appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally, all the PRs involved in breaking changes would be merged on the same day
to avoid breaking the folio-testing environment.
Communication is paramount if that is to be achieved,
especially as the number of inter-module and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems,
ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,6 +1,37 @@
...
+
+### Tech Debt
+* [MODINVOICE-522](https://folio-org.atlassian.net/browse/MODINVOICE-522) - Adding validation to exclude cases where the
+  'directory' field in batch group configuration consists solely of spaces
+
+### Dependencies
+* Bump `raml` from `35.0.1` to `35.2.0`
+* Bump `vertx` from `4.3.4` to `4.5.4`
+* Bump `mod-di-converter-storage-client` from `4.2.0-SNAPSHOT` to `4.2.0`
+* Bump `data-import-processing-core` from `2.1.0` to `2.1.8`
```
Comment (SerhiiNosko) [inline NEWS.md:33]: versions of data-import-processing-core and mod-di-converter-storage-client need to be swapped.
Basically if we updating from snapshot version to release version and version is the same - so no need to mention it in release notes

Comment (azizbekxm) [reply NEWS.md]: Thanks, updated

File: NEWS.md [hunk 2]
```diff
@@ -1,6 +1,36 @@
-## 5.8.0 - Unreleased
-* [MODSOURMAN-1022](https://issues.folio.org/browse/MODSOURMAN-1022) Remove step of initial saving of incoming records to SRS
+## 5.9.0 - Unreleased
+
+## 5.8.0 - Released (Quesnelia R1 2024)
```
Comment (SerhiiNosko) [inline NEWS.md:3]: I think we need to mention somehow that DB is added to mod-invoice module, maybe it would be usefull for dev-ops

Comment (azizbekxm) [reply NEWS.md:3]: added, thanks

**Review by SerhiiNosko — APPROVED** [review]

**Pull request merged**